src/Traits/HasRoles.php

```diff
@@ -224,7 +224,12 @@ public function hasRole($roles, string $guard = null): bool
             return false;
         }
 
-        return $roles->intersect($guard ? $this->roles->where('guard_name', $guard) : $this->roles)->isNotEmpty();
+        if ($roles instanceof Collection) {
+            return $roles->intersect($guard ? $this->roles->where('guard_name', $guard) : $this->roles)->isNotEmpty();
+        }
+
+        // Since simply returning `false` may be a lie.
```

Comment:
Remove the comment

Reply:
You must follow the guidelines of the package in which you want to collaborate, 
that's the reason why you don't get a second review
`InvalidArgumentException` already explains why is an exception, because it is not an abstract `Exception`

Reply:
>the status whether you're a member or collaborator or contributor.
I mean, as I don't know said status, I can't decide the importance of your review

that is really discriminatory and out of place, avoid that type of comment in the future
It seems to me that all collaborations can help, more eyes to review is better

Reply:
@drbyte Thanks for your response, I guess, this package has nowhere anything named `lastError()`, hence throwing is always the only option, hence the comment is not really required.

I will push it's removal then.